service/frontend/adminHandler.go

```diff
@@ -489,17 +489,27 @@ func (adh *AdminHandler) GetWorkflowExecutionRawHistoryV2(
 		return nil, adh.error(err, scope)
 	}
 
+	startEventID := pageToken.StartEventID + 1
+	endEventID := pageToken.EndEventID
```

Comment:
i would suggest remove these 2 variables, since startEventID looks like pageToken.StartEventID and the meaning of those are different, plz stick to pageToken.StartEventID and pageToken.EndEventID

Reply:
updated